realgud/common/track.el

```diff
@@ -193,6 +210,9 @@ evaluating (realgud-cmdbuf-info-loc-regexp realgud-cmdbuf-info)"
 	 (cmdbuf (or opt-cmdbuf (current-buffer)))
 	 )
     (unless (realgud:track-complain-if-not-in-cmd-buffer cmdbuf t)
+      (if (realgud:eval-command-p text)
+          (realgud:message-eval-results text))
+
```

Comment:
this function is getting too long but I'm not sure I want to take on a refactor just yet

Reply:
You are very correct that a refactor is much needed. I understand your trepidation. I have the same desire and trepidation.